frontend/frontend-react/src/components/template/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import './Header.css'
+import React, { Component } from 'react'
+
+export default class Main extends Component {
```

Comment:
You should consider @Lukazovic comments on [this PR](https://github.com/leo3210k/Project_Instagram/pull/4) regarding class/functional components, class toggle, and `<i />` usage alongside onClick etc.